ngx-build

```diff
@@ -314,7 +314,13 @@ sub apply_patches {
     chdir ".." or die "cannot switch to ..\n";
 
     chdir "nginx-$version" or die "cannot switch to nginx-$version\n";
-    shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
+    unless ($ver ge '001015008') {
+        shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_sess_cb_yield.patch");
+
+    } else {
```

Comment:
It's confusing to use `unless` with `else`. My brain is twisted. Please switch to `if` `else` instead.